pkg/modulereader/resreader_test.go

```diff
@@ -228,6 +229,144 @@ func (s *MySuite) TestGetInfo_MetaReader(c *C) {
 	c.Assert(err, ErrorMatches, expErr)
 }
 
+func (s *MySuite) TestUnmarshalOutputInfo(c *C) {
```

Comment:
You could have simplified your tests significantly if used just a string constains instead of `yaml.Node`

Reply:
Done.